### PR TITLE
Cleanup deploy script

### PR DESCRIPTION
**Pull request by saiyan86**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

This PR cleans up the script for deploying aks-engine clusters.

**Which issue(s) this PR fixes**:

N/A

**Release note:**
```
None
```

### CODE REVIEW

File: hack/deploy-cluster.sh [hunk 1]
```diff
@@ -163,75 +99,6 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi
...
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/deploy/rbac-csi-attacher.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/deploy/rbac-csi-driver-registrar.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/deploy/rbac-csi-provisioner.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/deploy/rbac-csi-snapshotter.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/deploy/csi-azurefile-provisioner.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/deploy/csi-azurefile-attacher.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/deploy/azurefile-csi-driver.yaml
-# create storage class.
-kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azurefile-csi-driver/master/deploy/example/storageclass-azurefile-csi.yaml
```
Comment (feiskyer) [inline hack/deploy-cluster.sh]: could we keep L166-177, so that AzureFile plugin would be deployed?

**Comment by feiskyer** [comment]
@saiyan86 thanks for the cleanup.

/ok-to-test

File: hack/deploy-cluster.sh (hunk 1 above)
Comment (saiyan86) [reply hack/deploy-cluster.sh]: They weren't removed: https://github.com/kubernetes/cloud-provider-azure/blob/2c7b86e50f55c4301ef87c54fbecd508ad05c106/hack/deploy-cluster.sh#L89-L100
They are line 89-100 in the new file.

Comment (feiskyer) [reply hack/deploy-cluster.sh]: this is for azuredisk, there should be another set of azurefile commands

**Comment by feiskyer** [comment]
@saiyan86  thanks, could you squash the commits?

**Pull request merged**